Documentation/en-us/Troubleshooting.md

```diff
@@ -0,0 +1,8 @@
+# Common Installation Issues
+
+Here are solutions to common issues that come up when using the framework.
+
+## No such module 'Quick'
+
+- Delete the _entire_ `~/Library/Developer/Xcode/DerivedData` direction, which includes `ModuleCache`.
+- Explicitly build (`Cmd+B`) the `Quick`, `Nimble`, and `Pods-ProjectNameTests` targets after enabled their schemes from the Manage Schemes dialog.
```

Comment:
Thanks! It worked for me, but I don't need to build `Quick` and `Nimble` separately. I've just build `Pods-ProjectNameTests` after enabled that scheme.